fairseq_utility.py

```diff
@@ -0,0 +1,78 @@
+import shlex
+import shutil
+import subprocess
+import json, os
+from pathlib import Path
+from collections import defaultdict, OrderedDict
```

Comment:
Please verify all the imports and if it's not used let's remove it.

Reply:
subprocess : used for visualising the child processes running while a call to the fairseq_train() function. 
json : Initially removed
os : used for accessing the working directories and retrieve the id of the processes that are running ( this helps us to see the total number of tokens visited, etc. )
Path : used to indicate that a google drive path to which we need to store the output files generated.
collections : the token vocabulary taken as an input through parameter in the above function defined ( "srcdict" ) is stored and used in thee form of a dictionary.